src/plugins/remark-modified-time.mjs

```
@@ -0,0 +1,28 @@
+//
+// export function remarkModifiedTime() {
+//   return function (tree, file) {
+//     const filepath = file.history[0];
+//     const result = execSync(`git log -1 --pretty="format:%cI" "${filepath}"`);
+//     file.data.astro.frontmatter.lastModified = result.toString();
+//   };
+// }
+
+import {statSync} from "fs";
+import getReadingTime from 'reading-time';
+import {toString} from 'mdast-util-to-string';
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)